src/openshiftCommonServices.module.js

```diff
@@ -128,26 +130,28 @@ hawtioPluginLoader.registerPreBootstrapTask(function(next) {
       API_DISCOVERY_ERRORS.push({
         data: data,
         textStatus: textStatus,
-        xhr: jqXHR
+        xhr: jqXHR,
+        fatal: true
       });
     });
 
+  // NOTE: This should only be used for development.
   // Additional servers can be defined for debugging and prototyping against new servers not yet served by the aggregator
   // There can not be any conflicts in the groups/resources from these API servers.
-  var additionalDeferreds = [];
-  _.each(window.OPENSHIFT_CONFIG.additionalServers, function(server) {
-   var baseURL = (server.protocol ? (server.protocol + "://") : protocol) + server.hostPort + server.prefix;
-   additionalDeferreds.push($.get(baseURL)
-    .then(_.partial(getGroups, baseURL, server), function(data, textStatus, jqXHR) {
-      if (server.required !== false) {
-        API_DISCOVERY_ERRORS.push({
-          data: data,
-          textStatus: textStatus,
-          xhr: jqXHR
-        });
-      }
-    }));
-  });
+  // var additionalDeferreds = [];
+  // _.each(window.OPENSHIFT_CONFIG.additionalServers, function(server) {
+  //  var baseURL = (server.protocol ? (server.protocol + "://") : protocol) + server.hostPort + server.prefix;
+  //  additionalDeferreds.push($.get(baseURL)
+  //   .then(_.partial(getGroups, baseURL, server), function(data, textStatus, jqXHR) {
+  //     if (server.required !== false) {
+  //       API_DISCOVERY_ERRORS.push({
+  //         data: data,
+  //         textStatus: textStatus,
+  //         xhr: jqXHR
+  //       });
+  //     }
+  //   }));
+  // });
```

Comment:
Any reason to hang onto this code?

Reply:
I can yank it. Wasn't sure if it would be useful in the future, but it's always in the git history.